src/backend/distributed/worker/worker_data_fetch_protocol.c

```diff
@@ -482,6 +482,7 @@ worker_apply_sequence_command(PG_FUNCTION_ARGS)
 	/* run the CREATE SEQUENCE command */
 	ProcessUtility(commandNode, commandString, PROCESS_UTILITY_TOPLEVEL,
 				   NULL, None_Receiver, NULL);
+	CommandCounterIncrement();
```

Comment:
Thanks for the thorough explanation :+1:, too bad this doesn't change behavior on 9.6 so we can't add a test for it.